proto/tendermint/abci/types.proto

```diff
@@ -168,6 +169,7 @@ message ResponseInfo {
 
   int64 last_block_height   = 4;
   bytes last_block_app_hash = 5;
+  uint32 last_core_chain_locked_height   = 100;
```

Comment:
According to this JIRA task we need to obtain masternode info on start and pass it to Tenderdash with the ResponseInfo response.
https://dashpay.atlassian.net/secure/RapidBoard.jspa?rapidView=101&modal=detail&selectedIssue=DP-2938
So I guess we need to be able to respond with the same fields for the individual node as are in the `ValidatorUpdate`? Specifically `proTxHash`, `pubKey` and `power`? In that case there should be an additional `validator_update` variable here in the ResponseInfo message.

Reply:
@Cofresi That is not correct. I might have given a slight incorrect demand to the platform team before. Basically the signer client (core) needs to be respond to the query for the proTxHash and the publicKey. Platform needs to do nothing. in that regard.

Reply:
Oh, that is good news! Happy to close the task mentioned above then on Monday